tsconfig.test.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    "removeComments": true,
+    "target": "ES5",
+    "module": "commonjs",
+    "noEmitOnError": false,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
+    "sourceMap": false,
+    "inlineSourceMap": true,
+    "jsx": "react"
+  },
+  "files": [
+    "typings/globals/backbone-global/index.d.ts",
+    "typings/globals/backbone/index.d.ts",
+    "typings/globals/enzyme/index.d.ts",
+    "typings/globals/jasmine/index.d.ts",
+    "typings/globals/jquery/index.d.ts",
+    "typings/globals/marionette/index.d.ts",
+    "typings/globals/react-dom/index.d.ts",
+    "typings/globals/react-redux/index.d.ts",
+    "typings/globals/react-tether/index.d.ts",
+    "typings/globals/react/index.d.ts",
+    "typings/globals/redux/index.d.ts",
+    "typings/globals/tether/index.d.ts",
+    "typings/globals/underscore/index.d.ts",
+    "typings/globals/webpack-env/index.d.ts",
+    "typings/index.d.ts",
```

Comment:
I think you should just have `"typings/index.d.ts"` instead of all the files in the `typings` folder